examples/transmission_main_example.py

```diff
@@ -150,16 +150,18 @@ def main(network, equipment, source, destination, sim_params, req=None):
     for dp_db in power_range:
         req.power = db2lin(pref_ch_db + dp_db)*1e-3
         if power_mode:
-            print(f'\nPropagating with input power = {lin2db(req.power*1e3):.2f}dBm :')
+            print(f'\nPropagating with input power = {ansi_escapes.cyan}{lin2db(req.power*1e3):.2f} dBm{ansi_escapes.reset}:')
         else:
-            print(f'\nPropagating in gain mode: power cannot be set manually')
-        infos = propagate2(path, req, equipment, show=len(power_range)==1)
+            print(f'\nPropagating in {ansi_escapes.cyan}gain mode{ansi_escapes.reset}: power cannot be set manually')
+        infos = propagate2(path, req, equipment)
+        if len(power_range) == 1:
+            for elem in path:
+                print(elem)
         if power_mode:
-            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f}dBm :')
+            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f} dBm:')
         else:
             print(f'\nTransmission results:')
-        #info message in gain mode
-        print(destination)
+        print(f'  Final GSNR (signal bw): {ansi_escapes.cyan}{mean(destination.snr):.02f} dB{ansi_escapes.reset}')
```

Comment:
The term `GSNR` is not used anywhere else in the code and is not a well established term as far as I know. In the transceiver output printing, `SNR total` is used. Let's at least use the same term in both outputs to avoid confusion.

Reply:
Thanks, I've added a commit which implements just that. In the longer term, I would like to switch to "GSNR" as this is what the Polito team has been using in their presentations for some time, but you're right that it is basically undefined right now.